psalm.xml

```diff
@@ -5,6 +5,7 @@
         xmlns="https://getpsalm.org/schema/config"
         xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
         errorLevel="1"
+        ensureOverrideAttribute="false"
```

Comment:
I'd prefer this was omitted and we added the attribute everywhere necessary. You can run `psalter --issues=MissingOverrideAttribute` followed by `phpcbf` to do this automatically.

Reply:
oh, we can't do that because the `Override` attribute exists since PHP 8.3, and this package should still support `8.1`.

Reply:
TIL 🙂